### PR TITLE
PY3 fixed scrapy bench command

### PR DESCRIPTION
**Pull request by kmike**

### CODE REVIEW

File: scrapy/utils/benchserver.py [hunk 1]
```diff
@@ -16,15 +16,15 @@ def render(self, request):
         total = _getarg(request, 'total', 100, int)
         show = _getarg(request, 'show', 10, int)
```
Comment (lopuhin) [inline scrapy/utils/benchserver.py]: I think ``_getarg`` might also need fixing - elements of ``request.args`` are bytes, and currently the values will always be at default values. At least it was in ``mockserver`` - https://github.com/scrapy/scrapy/pull/1692/files#diff-77ba10fc5f895eef2641f556c8ef6bc1L33

Comment (kmike) [inline scrapy/utils/benchserver.py]: a nice catch, just pushed a fix

**Comment by lopuhin** [comment]
Looks good! And the build passed twice, I hope this fixes unstable builds.

**Comment by eliasdorneles** [comment]
:+1:

**Pull request merged**